src/withErrorBoundary.tsx

```diff
@@ -26,7 +26,8 @@ export function withErrorBoundary<T extends IErrorBoundary>(
       error: null,
     };
 
-    public static displayName = displayName;
+    public static displayName =
```

Comment:
It's not the error boundary you are looking for.
Most, if not all, projects use [@stoplight/react-error-boundary](https://github.com/stoplightio/react-error-boundary) which already [supports](https://github.com/stoplightio/react-error-boundary/blob/master/src/withErrorBoundary.ts) the displayName correctly.

Reply:
If you have a component using the boundary exposed by ui-kit, simply convert it to use the new one.

Reply:
I'm coming here from `elements`, where this is being used still somewhere.

But if there's a separate repo for it, then why don't we:
1. Remove the duplicate implementation, make the `ui-kit` one an alias of the `react-error-boundary` one and deprecate it?
2. Remove the `ui-kit` boundary altogether in the next major.

Reply:
Yeah, setting an alias is OK, but you can also simply remove it.
I am aware this kind of a breaking change, but IIRC we don't use it anywhere else.